docs/modelling/index.rst

```diff
@@ -0,0 +1,8 @@
+Modelling
```

Comment:
Agreed. I have my spell checker set to en-US but apparently it is ok with both spellings (or spelings :smile:)... I will swap to on L